src/components/HomePage/Banner.js

```diff
@@ -74,6 +78,12 @@ const useStyles = makeStyles(theme => ({
 }));
 
 function renderInfo(classes) {
+	const tz = currentTimeZoneShort;
+	const month = monthFormatter.format(hothStart);
+	const startDay = hothStart.getDate();
+	const endDay = hothEnd.getDate();
+	const eventCrossesDate = startDay !== endDay;
```

Comment:
Maybe add a comment here that `startDay` and `endDay` are in the user's timezone?